frontend/app/src/bots/utils/botBar.tsx

```diff
@@ -2,8 +2,8 @@ import React, { useState } from 'react';
 import styled from 'styled-components';
 import { EuiGlobalToastList } from '@elastic/eui';
 import { Button } from '../../components/common';
-export default function BotBar(props: any) {
-  const { value, simple } = props;
+export default function BotBar(props: { value: string, simple?: any }) {
+  const { value } = props;
```

Comment:
if simple isnt used here lets just remove it

Reply:
fixed in [2afdc08](https://github.com/stakwork/sphinx-tribes/pull/533/commits/2afdc084f49a66b9d99ad46bb9ecbf295e3536cb)